scripts/cargo-install-all.sh

```diff
@@ -109,12 +109,14 @@ done
 (
   set -x
   # shellcheck disable=SC2086 # Don't want to double quote $rust_version
-  $cargo $maybeRustVersion build $maybeReleaseFlag "${binArgs[@]}"
+  $cargo $maybeRustVersion build $maybeReleaseFlag --bins
```

Comment:
qq: why is this needed? because of shellcheck?

Reply:
IIRC some bins were missing without